coderd/audit_test.go

```diff
@@ -279,6 +282,28 @@ func TestAuditLogsFilter(t *testing.T) {
 		})
 		require.NoError(t, err)
 
+		for _, resource := range workspace.LatestBuild.Resources {
+			t.Logf("Resource: %#v", resource)
+		}
+
+		// Create one log with "Connect"
+		err = client.CreateTestAuditLog(ctx, codersdk.CreateTestAuditLogRequest{
+			Action:       codersdk.AuditActionConnect,
+			ResourceType: codersdk.ResourceTypeWorkspaceAgent,
+			ResourceID:   workspace.LatestBuild.Resources[0].Agents[0].ID,
+			Time:         time.Date(2022, 8, 15, 14, 30, 45, 100, time.UTC), // 2022-8-15 14:30:45
+		})
+		require.NoError(t, err)
+
+		// Create one log with "Open"
+		err = client.CreateTestAuditLog(ctx, codersdk.CreateTestAuditLogRequest{
+			Action:       codersdk.AuditActionOpen,
+			ResourceType: codersdk.ResourceTypeWorkspaceApp,
+			ResourceID:   workspace.LatestBuild.Resources[0].Agents[0].Apps[0].ID,
+			Time:         time.Date(2022, 8, 15, 14, 30, 45, 100, time.UTC), // 2022-8-15 14:30:45
+		})
```

Comment:
You should give an `OrganizationID` since `ResourceRequiresOrgID` is true for these resources. I guess the endpoint to submit logs does not enforce this?

Reply:
This was missing from all entries in this test, so I added them for completeness.